### PR TITLE
Created a new model based on the most weighted features

### LINKED ISSUE #3: Improve the ML model based on different features as inputs and model classifier
The ML model in [model_build.ipynb](https://github.com/CMU-313/fall-2021-hw4-crabs-adjust-humidity/blob/master/dockerfile/apps/app.py) should have improved metrics beyond the baseline model provided. Different features should be experimented with, as well as different types of models. The resultant model should have a reasonably improved F1 score and should also include cross validation.

### PR DESCRIPTION
**Pull request by maggiehua01**
Modified the model ipynb file to find the top five most weighted features using sklearn's feature selection rather than randomly selecting three features. 

First separated the data into training and testing, and thus added cross validation with a test size of 0.3. Model was trained on the training data and then tested and evaluated on the testing data using the random forest classifier.

Results:
F1- 0.380952380952381
Precision- 0.4444444444444444
Recall- 0.3333333333333333
Accuracy- 0.7815126050420168

Resolves #3

### CODE REVIEW

**Review by SantianLin — APPROVED** [review]
💯

**Review by mtandon18 — APPROVED** [review]
Looks great! Amazing scores!
